### ISSUE TITLE
Mod Request: C2ME

### ISSUE THREAD

**Issue by ReperakPro** [zorroware/fabric-optimization-mods#43]
**Requirements for a mod to be added:**
1. The mod must be open-source.
2. The mod cannot sacrifice vanilla behavior for performance.
3. The mod must be actively maintained.

**Mod name:** C2ME

**Type of mod (e.g. rendering, lighting, extra):** worldgen, IO

**Repository (e.g. GitHub, GitLab, Gitea):** https://github.com/ishlandbukkit/c2me-fabric

**Developer/organization:** Ishland

**Additional notes:**
C2ME is mature enough to be considered stable. There's no glaring issues and is safe by default.

**Comment by mr-pi12** [comment]
Yeah, the latest releases are becoming increasingly compatible with more mods. I quite like c2me, but my pc apparently doesn't :/
